### PR TITLE
Update template automatically

### PR DESCRIPTION
**Pull request by Rich-Harris**
In theory, this will update https://github.com/sveltejs/kit-template-default every time a new version of `create-svelte` is released. This will allow us to use that repo as the base for things like https://node.new/sveltekit.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: packages/create-svelte/package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
 		"lint": "eslint --ignore-path .gitignore --ignore-path ../../.gitignore \"./*.js\" && npm run check-format",
 		"format": "npm run check-format -- --write",
 		"check-format": "prettier --check . --config ../../.prettierrc --ignore-path ../../.gitignore --ignore-path .gitignore --plugin prettier-plugin-svelte --plugin-search-dir=.",
-		"prepublishOnly": "npm run build"
+		"prepublishOnly": "npm run build",
+		"postpublish": "./scripts/update-template-repo.sh"
```
Comment (bluwy) [inline packages/create-svelte/package.json:37]:
```suggestion
		"postpublish": "bash ./scripts/update-template-repo.sh"
```

Do we need a leading `bash` command here? IIRC it's needed in Ubuntu (CI).

File: packages/create-svelte/scripts/update-template-repo.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  # $1 : relative filename
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+DIR=$(get_abs_filename $(dirname "$0"))
+TMP=$(get_abs_filename "$DIR/../node_modules/.tmp")
+
+echo "dir: $DIR"
+echo "tmp: $TMP"
+
```
Comment (Rich-Harris) [inline packages/create-svelte/scripts/update-template-repo.sh:13]: this was for testing, can be removed

```suggestion
```

File: packages/create-svelte/tsconfig.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
 		"checkJs": true,
 		"noEmit": true,
 		"noImplicitAny": true,
-		"target": "es2020",
```
Comment (benmccann) [inline packages/create-svelte/tsconfig.json:7]: this isn't necessarily a small change. everytime we've tweaked this it's had an impact on some platform and I wouldn't be surprised if it introduces some breakage somewhere. it'd probably be best to do in a separate PR with a changeset if it's necessary

Comment (Rich-Harris) [reply packages/create-svelte/tsconfig.json:7]: it doesn't matter anyway, this PR is arguably redundant since #3458 and the zillion follow-ups. going to close it

Comment (benmccann) [reply packages/create-svelte/tsconfig.json:7]: Ooh. @antony's PR made the same change though, which I don't feel great about. Is necessary or do you think we can back that part out?
